<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WRP Creator</string>
  </property>
  <widget class="QPushButton" name="CreateWRP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Create WRP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MapGridSize">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_MapGrid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Map Grid size</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_MapCell">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Map Cell size</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MapCellSize">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReadXYZ">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Read XYZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReadObjects">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Read objects</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ReadObjects">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveWRP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save WRP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MapLimit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_CellLimit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell Limit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CellLimit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_MapLimit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Map Limit</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_HeightLimit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height Limit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HeightLimit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RandomObjects">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Random Objects</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Offset">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Factor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Factor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Factor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>22</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Offset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Objects">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Objects:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TextureBMP">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Texture BMP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportObjects">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Objects</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Status">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>313</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
